crates/context_aware_config/src/db/schema.rs

```diff
@@ -601,6 +601,14 @@ diesel::table! {
 diesel::joinable!(default_configs -> functions (function_name));
 diesel::joinable!(dimensions -> functions (function_name));
 
+diesel::table! {
+    config_versions (id) {
+        id -> Text,
+        config -> Json,
```

Comment:
Currently if you don't do `NOT NULL`, it makes this
`config -> Nullable<Json>,`, Can you please check this?